### ISSUE TITLE
Swap stuck on step 5 (transferring cross chain)

### ISSUE THREAD

**Issue by Axxoul** [makoto/connext-test#4]
Swap is stuck on step 5. 

Console last message:
Waiting for transfer creation on channel 0x18F9b7Fa838B5A0E86a2bbcfD01c0D688D99cC18

URL used: http://offl1.surge.sh/#/exchanges/Quick-Honey/token/USDT

**Comment by Axxoul** [comment]
Also it seems funds are stuck, as they have disappeared from MATIC.

**Comment by ChristophSiegenthaler** [comment]
I now updated to the latest code (march-05) and can also get as far as step 5. Last console log in my case: Waiting for transfer creation on channel 0xe5061bB20CD0BA825EE3CB7D17FE578Dfd58e178
URL used: http://localhost:3000/#/exchanges/Honey-Quick/token/DAI

**Comment by makoto** [comment]
Yeah, actually I have been experiencing the same and in the middle of debugging. I will disable the swap button for now
